os/kernel/binary_manager/binary_manager.h

```diff
@@ -107,6 +138,8 @@ typedef struct statecb_node_s statecb_node_t;
 binmgr_bininfo_t *binary_manager_get_binary_data(uint32_t bin_idx);
 #define BIN_ID(bin_idx)                                 binary_manager_get_binary_data(bin_idx)->bin_id
 #define BIN_STATE(bin_idx)                              binary_manager_get_binary_data(bin_idx)->state
+#define BIN_RTTYPE(bin_idx)                             binary_manager_get_binary_data(bin_idx)->rttype
```

Comment:
How about changing BIN_TYPE?

Reply:
'type' is already used word for binary type like elf in binary manager.